src/chains/ethereum/ethereum/package.json

```diff
@@ -81,7 +81,6 @@
     "@types/fs-extra": "9.0.2",
     "@types/lodash.clonedeep": "4.5.6",
     "@types/secp256k1": "4.0.1",
-    "@types/seedrandom": "2.4.28",
```

Comment:
BTW, seems like this was fixed in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/51609

and IIRC how DefinitelyTypesd works, should be upgraded to `@types/seedrandom@3.0.0`